x/emissions/keeper/keeper_test.go

```diff
@@ -2192,10 +2203,17 @@ func (s *KeeperTestSuite) TestGetActiveTopics() {
 		Key:   nil,
 		Limit: 10,
 	}
-	activeTopics, _, err := keeper.GetIdsOfActiveTopics(ctx, pagination)
+	activeTopics, _, err := keeper.GetIdsActiveTopicAtBlock(ctx, 5, pagination)
 	s.Require().NoError(err, "Fetching active topics should not produce an error")
+	s.Require().Len(activeTopics, 1, "Should retrieve exactly two active topics")
 
-	s.Require().Len(activeTopics, 2, "Should retrieve exactly two active topics")
+	pagination = &types.SimpleCursorPaginationRequest{
+		Key:   nil,
+		Limit: 10,
+	}
+	activeTopics, _, err = keeper.GetIdsActiveTopicAtBlock(ctx, 15, pagination)
+	s.Require().NoError(err, "Fetching active topics should not produce an error")
+	s.Require().Len(activeTopics, 1, "Should retrieve exactly two active topics")
```

Comment:
In both cases the error message is somewhat misleading `"Should retrieve exactly two active topics"` because you are returning exactly 1 topic per block, but I get what you mean after staring at it -- there are 2 active topics overall